src/music-quiz.ts

```diff
@@ -206,6 +214,8 @@ export class MusicQuiz {
                     position = ':second_place:'
                 } else if (index === 3) {
                     position = ':third_place:'
+                } else if (index > 3) {
+                    position = index.'.'
```

Comment:
Oh I see... The 3 should just be a two :)

```suggestion
                } else if (index === 2) {
                    position = ':third_place:'
```